miden/tests/integration/cli/cli_test.rs

```diff
@@ -38,3 +40,40 @@ fn cli_run() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn cli_bundle_debug() {
+    let mut cmd = bin_under_test().command();
+    cmd.arg("bundle").arg("--debug").arg("./masm-examples/bundle/lib");
```

Comment:
The `masm-examples` directory is currently used as both example programs for new miden users, as well as benchmarking (see https://github.com/0xPolygonMiden/miden-vm/blob/b7ff6c842b64392738019eb49f2a8c99d6557568/miden/benches/program_execution.rs#L11-L66). So basically examples where people can learn a bit of syntax, as well as being worthy of benchmarking.

The bundle ones don't really qualify for that IMO, so I would put those next to this file (i.e. under `tests/integrating/cli/data/`)